main.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'app'
+
+def print_options
+  puts 'Choose an option:'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a student'
+  puts '4. Create a teacher'
+  puts '5. Create a book'
+  puts '6. Create a rental'
+  puts '7. List rentals for a given person id'
+  puts '8. Quit'
+end
+
+def handle_option(choice)
+  option_handler = {
+    1 => ListBooksHandler.new,
+    2 => ListPeopleHandler.new,
+    3 => CreateStudentHandler.new,
+    4 => CreateTeacherHandler.new,
+    5 => CreateBookHandler.new,
+    6 => CreateRentalHandler.new,
+    7 => ListRentalsForPersonHandler.new
+  }
+
+  handler = option_handler[choice]
+  handler&.handle || puts('Invalid option, please try again.')
+end
```

Comment:
- [ ] Please kindly make sure your console app can be running without any problem. When I run the app and chose any option in the menu I get an error, please kindly fix that 🙏🏾 🙏🏾 

<img width="976" alt="Screenshot 2023-07-27 at 14 18 05" src="https://github.com/musangamfure/school-library/assets/74501523/4fec417f-d2ae-46fa-b4d5-7d710c22593a">